modules/utxo-lib/src/bitgo/signature.ts

```diff
@@ -360,3 +370,41 @@ export function verifySignature(
 
   return signatureVerifications.length > 0 && signatureVerifications.every((v) => v.signedBy !== undefined);
 }
+
+export function signInput2Of3(
+  txBuilder: UtxoTransactionBuilder,
+  vin: number,
+  scriptType: ScriptType2Of3 | 'p2shP2pk',
+  pubkeys: Triple<Buffer>,
+  keyPair: bip32.BIP32Interface,
+  cosigner: Buffer,
+  amount: number
+): void {
+  let prevOutScriptType;
+  let redeemScript;
+  let witnessScript;
+  let witnessValue: number | undefined = amount;
+
+  if (isScriptType2Of3(scriptType)) {
+    prevOutScriptType = scriptType2Of3AsPrevOutType(scriptType);
+    ({ redeemScript, witnessScript } = createOutputScript2of3(pubkeys, scriptType));
```

Comment:
For taproot I think we'll want a redeem/tapscript value chosen by redeem index. These would both be undefined I think and tx builder wouldn't be able to sign without the tapscript if I'm following this correctly.

Reply:
instead of selecting by redeem index (which is a little opaque) my suggestion is to select the controlBlock/witnessScript by the cosigner argument: https://github.com/BitGo/BitGoJS/commit/5a4230795c58c8acebf0e636bc46124973c7ea4e